sdk/test-utils/recorder-new/MIGRATION.md

```diff
@@ -0,0 +1,160 @@
+  }
+  // Other options, e.g. sanitizers (which replace the customizationsOnRecordings option)
+});
+```
+
+And at the end of your test, stop the recorder:
+
+```ts
+await recorder.stop();
+```
+
+It is important that `recorder.stop()` is called, or otherwise the next test will throw an error when trying to start the already started recorder. Additionally, it is important that both the `start` and `stop` calls are awaited, for similar reasons.
+
+## Environment variables
+
+In the legacy recorder, the `replaceableVariables` option could be used to specify environment variables that would be replaced in the recording and set during playback. This could be used to ensure that secrets and user-specific options do not appear in the recording body.
+
+The Unified Recorder client provides this functionality through the use of the `envSetupForPlayback` option, which is passed when `recorder.start` is called. Like the legacy recorder, it takes in an object mapping environment variables to what they should be replaced with in the recording. For example:
+
+```ts
+await recorder.start({
+  envSetupForPlayback: {
+    TABLES_SAS_CONNECTION_STRING: "fakeConnectionString"
+  }
+});
+```
+
+Under the hood, this is powered by the Unified Recorder's sanitizer functionality.
+
+**⚠️Important:** To access environment variables, you must use the `env` export made available from the **new** recorder. This ensures that environment variables are sourced from the correct location (using `process.env` and `dotenv` in Node, and using `window.__env__` via Mocha in the browser), and also means that the environment variables set in `envSetupForPlayback` are used in playback mode.
+
+## Customizations on recordings
+
+A powerful feature of the legacy recorder was its `customizationsOnRecordings` option, which allowed for arbitrary replacements to be made to recordings. The new recorder's analog to this is the sanitizer functionality.
+
+For a simple find/replace, a `GeneralRegexSanitizer` can be used. For example:
+
+```ts
+recorder.addSanitizers({
+  generalRegexSanitizers: [
+    regex: "find",
+    value: "replace"
+  ],
+});
+```
+
+This example would replace all instances of `find` in the recording with `replace`.
+
+Other sanitizers for more complex use cases are also available.
+
+## AAD and the new `NoOpCredential`
+
+The new recorder does not record AAD traffic at present. As such, tests with clients using AAD should make use of the new `NoOpCredential` provided by the recorder library when in playback mode. For example:
+
+```ts
+const credential = isPlaybackMode()
+  ? new NoOpCredential();
+  : new ClientSecretCredential(/* ... */);
+```
+
+Since AAD traffic is not recorded by the new recorder, there is no longer a need to remove AAD credentials from the recording using a sanitizer.
+
+## Browser tests and modifications to Karma configuration
+
```

Comment:
Also remove this 
![image](https://user-images.githubusercontent.com/10452642/146350435-da57652e-20de-47af-a002-2a8081395469.png)
and related plugin